src/codacy-ameba/runner.cr

```diff
@@ -8,9 +8,15 @@ module Codacy::Ameba
     def run
       codacy_config = Config.load(dir)
 
-      ameba_config = ::Ameba::Config.load
-      configure_files(ameba_config, codacy_config.files)
-      configure_rules(ameba_config, codacy_config.tools)
+      ameba_config = ::Ameba::Config.load("#{dir}/.ameba.yml", true)
+
+      if codacy_config.is_a?(Codacy::Ameba::Config)
```

Comment:
here you are checking for `nil` so:
```
if codacy_config
```
is enough

Reply:
✅